IFComp/root/lib/site/html

```diff
@@ -75,8 +75,12 @@
         update_ui(do_dark);
         if (dark_mode_preferred == do_dark) {
             localStorage.removeItem('dark_mode');
+            // Set the cookie to be expired immediately
+            document.cookie = 'parchment_theme=light; domain=ifcomp.org; max-age=0';
```

Comment:
The site domain is hard coded here. I wasn't sure how to make this dependent on the site config.

Reply:
can you use `window.location.host` for the domain?

Reply:
Oh yeah, I had been thinking of a server side solution, but that would work.